### DISCUSSION TITLE
github.com settings

### DISCUSSION THREAD

**Discussion by IvBKo** [microsoft/mfcmapi#586] [General]
Hello,
I think there is some problem in the github.com settings - in the last 24 hours I have received more than 50 messages in my mail from you about changing the code. Sometimes every minute/two.
![2023-06-29](https://github.com/stephenegriffin/mfcmapi/assets/85292264/c052cb41-ae2a-4054-96cf-e68c516ac780)

**Comment by stephenegriffin** [comment]
You might wanna adjust your notification settings. I was rebuilding the build process and it took a few iterations to get it right.
